vm/package/deploy.rs

```diff
@@ -137,35 +137,40 @@ impl<N: Network> Package<N> {
     pub fn deploy<A: crate::circuit::Aleo<Network = N, BaseField = N::Field>>(
         &self,
         endpoint: Option<String>,
-    ) -> Result<Deployment<N>> {
```

Comment:
If we remove this `Deployment` output, what is the purpose of the case where `endpoint` is None.

When would the user tries to call `deploy` without an endpoint? The deployment is never generated, but the function runs to completion without issue.